### PR TITLE
kmeleonbrowser.org.K-Meleon version 75.1

### PR DESCRIPTION
**Pull request by techflashYT**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16387)

### CODE REVIEW

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml:3]:
```suggestion
PackageIdentifier: kmeleonbrowser.K-Meleon
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.yaml:3]:
```suggestion
PackageIdentifier: kmeleonbrowser.K-Meleon
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: kmeleonbrowser.K-Meleon
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
+PackageVersion: 75.1
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://versaweb.dl.sourceforge.net/project/kmeleon/k-meleon/75.1/K-Meleon75.1.exe
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml:8]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/kmeleon/files/k-meleon/75.1/K-Meleon75.1.exe/download
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
+PackageVersion: 75.1
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml:4]:
```suggestion
PackageVersion: 75.0
```
image

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
+PackageVersion: 75.1
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.yaml:4]:
```suggestion
PackageVersion: 75.0
```
image

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
+PackageVersion: 75.1
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 75.0
```
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml (hunk 4 above)
Comment (sanikdah) [reply manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml:8]: That isnt a direct download link, what i put is.

Comment (ItzLevvie) [reply manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml:8]: The one you put is a mirror, which will not always work for everyone.

Comment (sanikdah) [reply manifests/k/kmeleonbrowser/org/K-Meleon/75.1/kmeleonbrowser.org.K-Meleon.installer.yaml:8]: Alright it looks like it works anyway though.

**Comment by sanikdah** [comment]
I committed all the changes and now there's a manifest validation error?

**Comment by ItzLevvie** [comment]
You'll need to change the folders to match the `PackageIdentifier` using the GitHub Desktop client on Windows.

**Comment by sanikdah** [comment]
let me make the bot try again,

> You'll need to change the folders and file names to match the `PackageIdentifier` and `PackageVersion` using the GitHub Desktop client on Windows.

I had issues with that before, was I doing it wrong?  What I was doing is: fetch origin to make sure everything is up to date, view the repo files in file explorer, rename the folder, then doing CTRL+P to push the changes.  Is that incorrect?

**Comment by sanikdah** [comment]
image
is this what is supposed to happen when you rename 75.1 to 75.0?

**Comment by sanikdah** [comment]
There, I tried pushing the changes and it does nothing at all...

**Comment by sanikdah** [comment]
I'm not really sure what the issue is with the github desktop app or if it's just me doing something wrong.

**Comment by ItzLevvie** [comment]
> I had issues with that before, was I doing it wrong? What I was doing is: fetch origin to make sure everything is up to date, view the repo files in file explorer, rename the folder, then doing CTRL+P to push the changes. Is that incorrect?

Yes, you need to rename the folder and then commit the changes and then Ctrl + P to push the changes.

You need to remove the `org` folder and move everything that's inside of that folder to the root of `kmeleonbrowser` and rename the version folder to `75.0`

**Comment by sanikdah** [comment]
> > I had issues with that before, was I doing it wrong? What I was doing is: fetch origin to make sure everything is up to date, view the repo files in file explorer, rename the folder, then doing CTRL+P to push the changes. Is that incorrect?
> 
> Yes, you need to rename the folder and then commit the changes (write a commit message, etc.) and then Ctrl + P to push the changes to your branch.
> 
> You need to remove the `org` folder and move everything that's inside of that folder to the root of `kmeleonbrowser` and rename the version folder to `75.0`

Alright I can do all of that except for commit, the button is grayed out and clicking it does nothing.
image

**Comment by ItzLevvie** [comment]
Sorry, I mean write a commit summary (not the description), the button shows up for me when all of the files are ticked and a summary is written.

image

**Comment by sanikdah** [comment]
Sorry, I didn't even notice those 🤦

**Comment by sanikdah** [comment]
Yay!!

**Comment by sanikdah** [comment]
But....

**Comment by sanikdah** [comment]
Did I do it wrong somehow?

**Comment by ItzLevvie** [comment]
Sorry, you also need to rename the file names too to match `PackageIdentifier`

**Comment by sanikdah** [comment]
There we go :)

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**